README.md

```diff
@@ -187,6 +187,45 @@ Finally,  re-run the project playbook, limiting the scope to the machine that yo
 $ OULIB_USER=vagrant vagrant ssh ansible --command "ansible-playbook /vagrant/projects/web/playbooks/vagrant.yml --limit solr.vagrant.localdomain"
 ```
 
+Ad-hoc Ansible Commands
+------
+As this is an attempt to replicate the production enviornment as closely as possible ad-hoc Ansible commands are not immediately available and require one to SSH into a specific vagrant box within the prod-infrastructure. Below are two scenarios:
+
+### Scenario #1:
```

Comment:
I think I'd leave this non-working example out. If I were skimming the docs, I'm not sure that I'd notice that it was an example of what not to do until after I'd already started doing the steps.